Automattic/jetpack
# CRM: Moving the invoice status selector underneath the ID

## Proposed changes:

* This PR moves the Invoice Status options from the Invoice Action Metabox to underneath the Invoice ID (on the Invoice Edit page).
* The process was a little different to that used when moving the Quote status ([PR here](https://github.com/Automattic/jetpack/pull/29653)), as the Invoice edit page details here are output via JS instead of PHP.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2949

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this, using the Jetpack Beta plugin on a test site with this branch applied or testing locally with the branch checked out, visit the 'Edit' Invoice page for an Invoice: `/wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=invoice&zbsid={quoteid}`. 
* Underneath the Invoice ID you should see a Status label, with a dropdown for the available statuses.
* Try changing the status and saving - the change should save and the status should be visible on the 'View All' Invoices page (plus in the header of the 'edit Invoice' page).
* The Status dropdown should no longer be visible in the Invoice Actions metabox.
